src/js/pagination/paging-collection.js

```diff
@@ -83,7 +83,16 @@
              * backbone.paginator.
              */
             parseState: function (response, queryParams, state, options) {
-                return {totalRecords: response[0].count, totalPages: response[0].num_pages};
+                var newState = _.clone(state);
+                newState.totalRecords = response[0].count;
+                newState.totalPages = response[0].num_pages;
+
+                // Remove already processed response
+                delete response.count;
+                delete response.num_pages;
+
+                // Update the state with response
+                return PageableCollection.prototype.parseState.call(this, response, queryParams, newState, options);
             },
```

Comment:
Hi @ehteshamkafeel - Sorry for the lack of communication. I should have updated you the other day. I've been busy with development and code reviews for the Analytics team. I'm hoping to re-review this afternoon or on Monday.